docs/source/cloud-setup/policy.rst

```diff
@@ -346,3 +347,21 @@ Use local GCP credentials for all tasks
 .. note::
 
     This policy only take effects when applied at :ref:`client-side <client-side-admin-policy>`. Use this policy at the :ref:`server-side <server-side-admin-policy>` will be a no-op.
+
+.. _add-volumes-policy:
+
+Add volumes to all tasks
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. literalinclude:: ../../../examples/admin_policy/example_policy/example_policy/client_policy.py
+    :language: python
+    :pyobject: AddVolumesPolicy
+    :caption: `AddVolumesPolicy <https://github.com/skypilot-org/skypilot/blob/master/examples/admin_policy/example_policy/example_policy/client_policy.py>`_
+
+.. literalinclude:: ../../../examples/admin_policy/add_volumes.yaml
+    :language: yaml
+    :caption: `Config YAML for using AddVolumesPolicy <https://github.com/skypilot-org/skypilot/blob/master/examples/admin_policy/add_volumes.yaml>`_
+
+.. note::
+
+    This policy only take effects when applied at :ref:`client-side <client-side-admin-policy>`. Use this policy at the :ref:`server-side <server-side-admin-policy>` will be a no-op.
```

Comment:
Why is this so? Would this not update the volume mounts even if it's set at server side?

Reply:
Updated to apply to both client and server sides.